packages/shared/src/components/Icon.tsx

```diff
@@ -37,6 +38,7 @@ const Icon = ({
   return (
     <IconComponent
       className={classNames(IconSize[size], 'pointer-events-none', className)}
+      style={size === 'large' ? { fontSize: '1.75rem' } : null}
```

Comment:
I think it is confusing to have auto values but then set a specific styling value. We need to make it consistent across different sizes.

Reply:
Yeah, it was a hack way of adding a new size just because h-7 w-7 wasn't working for me (next.js cache 😅), now it's good

Reply:
Ah, yes. We have this ongoing issue with nextjs - we're hoping to solve it soon enough. If in case the thing we're working on is applicable in extension as well, personally I would suggest testing things in there so you won't have to clear cache and re-run every time.